src/key-value-pairs/__tests__/key-value-pairs.test.tsx

```diff
@@ -0,0 +1,136 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import createWrapper from '../../../lib/components/test-utils/dom';
+import KeyValuePairs, { KeyValuePairsProps } from '../../../lib/components/key-value-pairs';
+
+function renderKeyValuePairs(props: KeyValuePairsProps) {
```

Comment:
We prefer to accept JSX here, not the props object:

```js
renderKeyValuePairs(<KeyValuePairs columns={...} />)
```

This is not much more to write, but the tests become also usage examples, which is nice to have

Reply:
updated